test/Utilities/Microsoft.Testing.TestInfrastructure/CommandLine.cs

```diff
@@ -72,7 +72,7 @@ public async Task<int> RunAsyncAndReturnExitCodeAsync(
         IDictionary<string, string?>? environmentVariables = null,
         string? workingDirectory = null,
         bool cleanDefaultEnvironmentVariableIfCustomAreProvided = false,
-        int timeoutInSeconds = 60)
+        int timeoutInSeconds = int.MaxValue)
```

Comment:
Looks like this may not work (when multiplied by 1000 to convert to milliseconds, it crosses `Timer.MaxSupportedTimeout`.

Switch instead to just simply a big enough value, like 10000. This applies to all timeouts changed in the PR.